app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,10 @@
+class FoodsController < ApplicationController
+  def index
+    @foods = FoodsFacade.top_10_results(query_params)
+    @count = FoodsFacade.foods_count(query_params)
```

Comment:
Both of these are hitting the same endpoint. I'll leave a comment in the facade with an example of how you could get both of these things with one API call and assign them like this:
`@foods, @count = FoodsFacade.search_results(query_params)`